docs/get-started/vue-vite.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Storybook for Vue & Vite
+---
+
+Storybook for Vue & Vite is a [framework](../contribute/framework.md) that makes it easy to develop and test UI components in isolation for [Vue](TK) applications built with [Vite](TK). It includes:
+
+- TK - Emoji feature list
+- 💫 and more!
+
+## Requirements
+
+- Vue >= 3
+- Vite >= 3.0 (4.X recommended)
+- Storybook >= 7.0
+
+## Getting started
+
+### In a project without Storybook
+
+Follow the prompts after running this command in your Next.js project's root directory:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'common/init-command.npx.js.mdx',
+   'common/init-command.yarn.js.mdx',
+   'common/init-command.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+[More on getting started with Storybook.](./install.md)
+
+### In a project with Storybook
+
+This framework is designed to work with Storybook 7. If you’re not already using v7, upgrade with this command:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-upgrade.npm.js.mdx',
+    'common/storybook-upgrade.pnpm.js.mdx',
+    'common/storybook-upgrade.yarn.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+#### Automatic migration
+
+When running the `upgrade` command above, you should get a prompt asking you to migrate to `@storybook/vue3-vite`, which should handle everything for you. In case that auto-migration does not work for your project, refer to the manual migration below.
+
+#### Manual migration
+
+First, install the framework:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'vue/vue3-vite-install.npm.js.mdx',
+    'vue/vue3-vite-install.pnpm.js.mdx',
+    'vue/vue3-vite-install.yarn.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Then, update your `.storybook/main.js|ts` to change the framework property:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'vue/vue3-vite-add-framework.js.mdx',
+    'vue/vue3-vite-add-framework.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Extending the Vue application
+
+Storybook creates a [Vue 3 application](https://vuejs.org/api/application.html#application-api) for your component preview.
+When using global custom components (`app.component`), directives (`app.directive`), extensions (`app.use`), or other application methods, you will need to configure those in the `./storybook/preview.js` file.
+
+Therefore, Storybook provides you with a `setup` function exported from this package, which receives as a callback your Storybook instance, which you can interact with and add your custom configuration.
+
+```js
+// .storybook/preview.js
+import { setup } from '@storybook/vue3';
+
+setup((app) => {
+  app.use(MyPlugin);
+  app.component('my-component', MyComponent);
+  app.mixin({
+    /* My mixin */
+  });
+});
+
+// Rest of the file...
+```
+
+## Using `vue-component-meta`
```

Comment:
Should we also be transparent about current Volar limitations? Something like:

Due to a [vue-component-meta limitation](https://github.com/vuejs/language-tools/issues/3896), `tsconfig.json` files that include `references` are currently not supported. This can lead to missing resolved types/controls if you are using import aliases (`compilerOptions.paths` in your tsconfig).

Until the feature request is implemented in `vue-component-meta`, you can switch to relative imports for types used for props, emits, slots or exposed:
```ts
import type { MyProps } from "@/types"; // can not be resolved
import type { MyProps } from "../types"; // can be resolved
```